ui/admin/tests/acceptance/aliases/update-test.js

```diff
@@ -78,4 +93,21 @@ module('Acceptance | aliases | update', function (hooks) {
     assert.dom(NAME_FIELD_SELECTOR).hasValue(`${name}`);
     assert.strictEqual(instances.alias.name, name);
   });
+
+  test('users have the option to clear an alias', async function (assert) {
```

Comment:
It probably would be good to have a quick test that the user can't clear without update permissions but this is not blocking

Reply:
agree, added!